lighthouse-core/runner.js

```diff
@@ -144,7 +144,8 @@ class Runner {
           score = report.score;
         }
 
-        return {
+        return opts.driver.getChromeVersion().then(version => ({
```

Comment:
actually, we do need it in gather-runner, as runner doesn't necessarily have a driver or a browser to talk to. I'd be ok with a `ChromeVersion` artifact on there.

Is there value in saving the whole UA string? e.g operating system might be helpful.